From 8334a73c00fa01147efa200a1c6bde525639a8a6 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Tue, 18 Mar 2014 09:18:45 -0700
Subject: [PATCH 489/827] media: xilinx: tpg: Fix alignments

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Fix minor alignments.

Signed-off-by: Hyun Kwon <hyunk@xilinx.com>
Acked-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit a6610f93128f8efa321cdb990878b3fda06b35db)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-tpg.c | 40 +++++++++++++++---------------
 1 file changed, 20 insertions(+), 20 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-tpg.c b/drivers/media/platform/xilinx/xilinx-tpg.c
index c3c21b8..012620a 100644
--- a/drivers/media/platform/xilinx/xilinx-tpg.c
+++ b/drivers/media/platform/xilinx/xilinx-tpg.c
@@ -159,8 +159,8 @@ static int xtpg_s_stream(struct v4l2_subdev *subdev, int enable)
  */
 
 static int xtpg_enum_mbus_code(struct v4l2_subdev *subdev,
-				     struct v4l2_subdev_fh *fh,
-				     struct v4l2_subdev_mbus_code_enum *code)
+			       struct v4l2_subdev_fh *fh,
+			       struct v4l2_subdev_mbus_code_enum *code)
 {
 	struct xtpg_device *xtpg = to_tpg(subdev);
 
@@ -173,8 +173,8 @@ static int xtpg_enum_mbus_code(struct v4l2_subdev *subdev,
 }
 
 static int xtpg_enum_frame_size(struct v4l2_subdev *subdev,
-				      struct v4l2_subdev_fh *fh,
-				      struct v4l2_subdev_frame_size_enum *fse)
+				struct v4l2_subdev_fh *fh,
+				struct v4l2_subdev_frame_size_enum *fse)
 {
 	struct xtpg_device *xtpg = to_tpg(subdev);
 
@@ -204,8 +204,8 @@ __xtpg_get_pad_format(struct xtpg_device *xtpg, struct v4l2_subdev_fh *fh,
 }
 
 static int xtpg_get_format(struct v4l2_subdev *subdev,
-				 struct v4l2_subdev_fh *fh,
-				 struct v4l2_subdev_format *fmt)
+			   struct v4l2_subdev_fh *fh,
+			   struct v4l2_subdev_format *fmt)
 {
 	struct xtpg_device *xtpg = to_tpg(subdev);
 
@@ -215,8 +215,8 @@ static int xtpg_get_format(struct v4l2_subdev *subdev,
 }
 
 static int xtpg_set_format(struct v4l2_subdev *subdev,
-				 struct v4l2_subdev_fh *fh,
-				 struct v4l2_subdev_format *fmt)
+			   struct v4l2_subdev_fh *fh,
+			   struct v4l2_subdev_format *fmt)
 {
 	struct xtpg_device *xtpg = to_tpg(subdev);
 	struct v4l2_mbus_framefmt *__format;
@@ -348,10 +348,10 @@ static struct v4l2_subdev_video_ops xtpg_video_ops = {
 };
 
 static struct v4l2_subdev_pad_ops xtpg_pad_ops = {
-	.enum_mbus_code = xtpg_enum_mbus_code,
-	.enum_frame_size = xtpg_enum_frame_size,
-	.get_fmt = xtpg_get_format,
-	.set_fmt = xtpg_set_format,
+	.enum_mbus_code		= xtpg_enum_mbus_code,
+	.enum_frame_size	= xtpg_enum_frame_size,
+	.get_fmt		= xtpg_get_format,
+	.set_fmt		= xtpg_set_format,
 };
 
 static struct v4l2_subdev_ops xtpg_ops = {
@@ -361,8 +361,8 @@ static struct v4l2_subdev_ops xtpg_ops = {
 };
 
 static const struct v4l2_subdev_internal_ops xtpg_internal_ops = {
-	.open = xtpg_open,
-	.close = xtpg_close,
+	.open	= xtpg_open,
+	.close	= xtpg_close,
 };
 
 /*
@@ -696,13 +696,13 @@ static const struct of_device_id xtpg_of_id_table[] = {
 MODULE_DEVICE_TABLE(of, xtpg_of_id_table);
 
 static struct platform_driver xtpg_driver = {
-	.driver = {
-		.owner = THIS_MODULE,
-		.name = "xilinx-axi-tpg",
-		.of_match_table = of_match_ptr(xtpg_of_id_table),
+	.driver			= {
+		.owner		= THIS_MODULE,
+		.name		= "xilinx-axi-tpg",
+		.of_match_table	= of_match_ptr(xtpg_of_id_table),
 	},
-	.probe = xtpg_probe,
-	.remove = xtpg_remove,
+	.probe			= xtpg_probe,
+	.remove			= xtpg_remove,
 };
 
 module_platform_driver(xtpg_driver);
-- 
2.9.3

